dimik-even-odd-1/en.md

```diff
@@ -1,6 +1,6 @@
 # Dimik - Even Odd 1
 
-In this problem, you will be given `T` testcases.Each line of the testcase consist of an integer `n`.We have to determine whether `n` is even or odd.
+In this problem, you will be given `T` testcases.Each line of the testcase consists of an integer `n`.We have to determine whether `n` is even or odd.
```

Comment:
While we are at it, let's also put one space after each line. So after each full stop denoting a line end, put an space before you start the next line in the same paragraph.